testsuite/testsuite-jsp/pom.xml

```diff
@@ -49,6 +64,12 @@
             <version>${project.version}</version>
             <executions>
               <execution>
+                <id>package</id>
```

Comment:
@kenfinnigan Can you explain what you with profile `uberjar `? I don't any such thing in the testsuite modules. There is a profile `uberjar ` in standalone-servers. Am I missing something?